python_modules/dagster/dagster/_core/definitions/automation_tick_evaluation_context.py

```diff
@@ -230,6 +230,9 @@ def _flood_fill_asset_subsets(k: EntityKey):
         visited.add(k)
         if isinstance(k, AssetKey):
             node = asset_graph.get(k)
+            if not node.is_materializable:
+                # if the asset is not materializable, we should not backfill it
```

Comment:
nit: maybe "it cannot be included in a backfill" instead of "we should not backfill it"